router/router.go

```diff
@@ -79,6 +84,7 @@ type HandleT struct {
 	failedEventsListMutex sync.RWMutex
 	failedEventsList      *list.List
 	failedEventsChan      chan jobsdb.JobStatusT
+	readProcessChan       chan readProcessStruct
```

Comment:
we are done reading, just processing now :)
```suggestion
	processChan       chan jobsWithLatencies
```